test/test_prototype_utils.py

```diff
@@ -1,5 +1,5 @@
 import pytest
-from torchvision.prototype.utils._internal import sequence_to_str
```

Comment:
We should rename this whole file to `test_internal_utils` or the like. Otherwise, the test will only be run by the prototype tests.